packages/react-composites/src/composites/ChatComposite/file-sharing/UploadedFile.tsx

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import { EventEmitter } from 'events';
+
+/**
+ * Meta Data containing information about the uploaded file.
+ * @beta
+ */
+export interface FileMetaData {
+  /**
+   * File name to be displayed.
+   */
+  name: string;
+  /**
+   * Extension is used for rendering the file icon.
+   * An unknown extension will be rendered as a generic icon.
+   * Example: `jpeg`
+   */
+  extension: string;
```

Comment:
If we only render a known list of extensions, we should use a string literal type here `extension?: 'jpeg' | 'png'`

Reply:
The list is huge. And is dependent on the underlying Fluent UI.
Even Fluent UI doesn't have a documented list of supported icons.